roles/fail2ban-iptables/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+# you might set it to INFO on service with low traffic or to monitor more closely
+fail2ban_iptables_loglevel: NOTICE
+fail2ban_iptables_ignoreips: >-
```

Comment:
The fail2ban_iptables_ignoreips setting is using organization-specific variables:
  - off_internal_hosts_ips_v4
  - off_private_networks_v4
  - off_internal_hosts_ips_v6
  - off_private_networks_v6

  These variables (with the "off_" prefix) are clearly specific to Open Food Facts.

  For a truly reusable role:
  1. The role should use generic variable names
  2. It should not assume organization-specific variables exist
  3. It should provide sensible defaults that work in any environment

Reply:
I will move it in the role.